st2client/st2client/commands/pack.py

```diff
@@ -190,12 +190,41 @@ def __init__(self, resource, *args, **kwargs):
                                  help='Force pack installation.')
 
     def run(self, args, **kwargs):
+        if args.packs:
+            if len(args.packs) == 1:
```

Comment:
Perhaps we can utilize this in a separate utility function or similar so it's easier to test it.